core/templates/domain/improvements/needs-guiding-response-task.model.ts

```diff
@@ -56,11 +56,11 @@ export class NeedsGuidingResponsesTask extends TaskEntry<
       task_type: ImprovementsConstants.TASK_TYPE_NEEDS_GUIDING_RESPONSES,
       target_type: ImprovementsConstants.TASK_TARGET_TYPE_STATE,
       target_id: stateName,
-      issue_description: null,
+      issue_description: '',
       status: ImprovementsConstants.TASK_STATUS_OBSOLETE,
-      resolver_username: null,
-      resolver_profile_picture_data_url: null,
-      resolved_on_msecs: null,
+      resolver_username: '',
+      resolver_profile_picture_data_url: '',
+      resolved_on_msecs: undefined,
```

Comment:
Let's keep `null` and allow it as a value.

Reply:
Done

Reply:
What I meant is to keep `null` on all the lines that I mentioned.

Reply:
Done